aviary/subsystems/propulsion/prop_performance.py

```diff
@@ -0,0 +1,244 @@
+import openmdao.api as om
+import numpy as np
+from dymos.models.atmosphere import USatm1976Comp
+from aviary.constants import TSLS_DEGR
+from aviary.utils.aviary_values import AviaryValues
+from aviary.variable_info.variables import Aircraft, Dynamic
+from aviary.subsystems.propulsion.hamilton_standard import HamiltonStandard, PostHamiltonStandard, PreHamiltonStandard
+
+
+class InstallLoss(om.Group):
+    """
+    Compute installation loss
+    """
+
+    def initialize(self):
+        self.options.declare(
+            'num_nodes', types=int,
+            desc='Number of nodes to be evaluated in the RHS')
+        self.options.declare(
+            'aviary_options', types=AviaryValues,
+            desc='collection of Aircraft/Mission specific options')
+
+    def setup(self):
+        nn = self.options['num_nodes']
+        self.add_subsystem(
+            name='sqa_comp',
+            subsys=om.ExecComp(
+                'sqa = minimum(DiamNac**2/DiamProp**2, 0.32)',
+                DiamNac={'units': 'ft'},
+                DiamProp={'units': 'ft'},
+                sqa={'units': 'unitless'},
+            ),
+            promotes_inputs=[("DiamNac", Aircraft.Nacelle.AVG_DIAMETER),
+                             ("DiamProp", Aircraft.Engine.PROPELLER_DIAMETER)],
+            promotes_outputs=["sqa"],
+        )
+
+        # We should update these minimum calls to use a smooth minimum so that the
+        # gradient information is C1 continuous.
+        self.add_subsystem(
+            name='zje_comp',
+            subsys=om.ExecComp(
```

Comment:
has_daig_partials=True for vectorized comps.

Reply:
Good call, added in all five places